src/frontend/src/lib/components/icons/IconBackArrow.svelte

```diff
@@ -0,0 +1,19 @@
+<!-- source: ISC Lucide - please visit https://lucide.dev/license -->
```

Comment:
Unrelated: if we have a sub-folder `iconly`, should we have a `lucide` subfolder under icons?

Reply:
yes, makes sense!

Reply:
done:

- PR #2942